_drafts/release-workflows.md

```diff
@@ -0,0 +1,388 @@
+| **push** to `master`                                | Release on default channel (if supported, otherwise no channel)                                                  |
+| **merge** feature branch to `master`                | Release on default channel (if supported, otherwise no channel)                                                  |
+| **push** to `next`                                  | -                                                                                                                |
+| **push** to `1.x.x` or `2.x.x` branches             | Release on default channel (if supported, otherwise no channel) if in range, report an error and error otherwise |
+| **merge** any branch to `1.x.x` or `2.x.x` branches | Release on default channel (if supported, otherwise no channel) if in range, report an error and error otherwise |
+
+Note: By default lts releases are published on the default channel as they are limited to a range of versions inferior to the versions released from `master`. Dependents will get only the expected releases by specifying a range dependency (for example `^2.0.0`).
+If a `channel` is specified, then the release will be made on this channel.
+
+##### LTS branches configuration
+
+```json
+"release": {
+  "branches": ["1.x.x", "2.x.x", "master"]
+  }
+}
+```
+
+With specific channel names:
+
+```json
+"release": {
+  "branches": [{
+    "branch": "v1",
+    "range": "1.x.x",
+    "channel": "v1"
+  }, {
+    "branch": "v2",
+    "range": "2.x.x",
+    "channel": "v2"
+  }, "master"]
+}
+```
+
+#### Pre-releases
+
+Workflow to distribute releases without incrementing the semantic version during the development of new a application version.  
+Ideal for applications distributing unstable/alpha/preview releases.
+
+Branch: `master`, `next-beta`, `next-alpha`
+
+Channel: `latest` if supported by the release target
+
+##### Pre-releases workflow
+
+| Action                                                         | Result                                                                                                                                        |
+|----------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------|
+| **push** to `master`                                           | Release on default channel (if supported, otherwise no channel)                                                                               |
+| **merge** `4.0.0-beta`, `5.0.0-beta` or any branch to `master` | Release on default channel (if supported, otherwise no channel)                                                                               |
+| **push** to `4.0.0-beta`                                       | Release a prerelease version (`4.0.0-beta`, then `4.0.0-beta.1`, then `4.0.0-beta.2`) on default channel (if supported, otherwise no channel) |
```

Comment:
is publishing on `@latest` what you want here? This means that people will get the `4.0.0-beta` version when they run `npm install your-package`. Maybe the channel should be `@beta` by default, the string after `4.0.0-<channel name>`?

Reply:
That's indeed wrong. It would be published in `@<preif>` channel. I'll fix it